nitric/api/documents.py

```diff
@@ -244,24 +250,28 @@ def to_collection_ref(self):
         return CollectionRef(
             self._documents,
             self.name,
-            DocumentRef(
+            self.parent
+            if self.parent is None
+            else DocumentRef(
                 self._documents,
                 self.parent,
                 NIL_DOC_ID,
             ),
         )
 
     @staticmethod
-    def from_collection_ref(collectionRef: CollectionRef, documents: Documents) -> CollectionGroupRef:
+    def from_collection_ref(collectionRef: CollectionRef, documents: Documents) -> Union[CollectionGroupRef, None]:
         """Return a collection ref as a collection group."""
         if collectionRef.parent is not None:
             return CollectionGroupRef(
                 documents,
                 collectionRef.name,
-                CollectionGroupRef.from_collection_ref(
-                    collectionRef.parent,
-                    documents,
-                ),
+                collectionRef.parent.parent,
+                # FIXME: Need to determine why this was here
+                # CollectionGroupRef.from_collection_ref(
+                #     collectionRef.parent.parent,
+                #     documents,
+                # ),
```

Comment:
Just want to make a note of this one to test collections once this change is in RC